Gemfile

```diff
@@ -30,6 +30,8 @@ gem 'pmp', '~> 0.5'
 
 gem 'hyperresource'
 
+gem 'addressable'
```

Comment:
The fragment is not transmitted to the server with the request.
On Jan 26, 2015 2:00 PM, "Andrew Kuklewicz" <notifications@github.com>
wrote:

> In Gemfile
> <https://github.com/PRX/pmp-importer/pull/9#discussion_r23554871>:
>
> > @@ -30,6 +30,8 @@ gem 'pmp', '~> 0.5'
> >
> >  gem 'hyperresource'
> >
> > +gem 'addressable'
>
> I am building a feed server that checks the fragment. [image:
> :smiling_imp:]
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/PRX/pmp-importer/pull/9/files#r23554871>.
>